services/terrain/src/terrain/routes/fileio.clj

```diff
@@ -10,8 +10,8 @@
   []
   (util/optional-routes [config/data-routes-enabled]
 
-    (GET "/fileio/download" [:as {:keys [params]}]
-      (fio/download params))
+    (GET "/fileio/download" [:as req]
+      (util/controller req fio/download :params))
```

Comment:
This construction using `terrain.util/controller` matches the things we're doing in `routes.filesystem`, which seemed easier than discerning exactly what differences mattered and didn't.